firmware/defmt-test/README.md

```diff
@@ -110,16 +108,23 @@ mod tests {
 
 ``` console
 $ cargo test -p testsuite
-0.000000 INFO  running assert_true ..
-└─ test::tests::__defmt_test_entry @ tests/test.rs:11
-0.000001 INFO  .. assert_true ok
+0.000000 INFO  (1/2) running `assert_true`...
 └─ test::tests::__defmt_test_entry @ tests/test.rs:11
-0.000002 INFO  running assert_flag ..
+0.000001 INFO  (2/2) running `assert_flag`...
 └─ test::tests::__defmt_test_entry @ tests/test.rs:11
-0.000003 INFO  .. assert_flag ok
+0.000002 INFO  all tests passed!
 └─ test::tests::__defmt_test_entry @ tests/test.rs:11
 ```
 
+## Test Outcome
```

Comment:
:100: 
perhaps we should an example of this to app-template? it's a bit of 'duplicated documentation' but I tend to favor 'put docs right in front of the user'; that way 'didn't read the docs' issues become less common (hopefully)